src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -85,8 +85,15 @@ interface Counts {
 // Possibly just extend `BaseFilterContext` and make the former ones into `BaseFilterContext<ArtworkFilters>`
 // and `BaseFilterContext<AuctionResultFilters>`.
 export interface ArtworkFilterContextProps {
+  /** The current artwork filter state (which determines the network request and the url querystring) */
   filters?: ArtworkFilters
 
+  /** Interim filter state, to be manipulated before being applied to the current filter state */
+  stagedFilters?: ArtworkFilters
```

Comment:
This represents the object that holds the temporary state that gets manipulated while the mobile sheet is open.